### PR TITLE
docs: add rbac example setup diagram

### PR DESCRIPTION
**Pull request by tara-det-ai**
add example setup diagram to help illustrate usage

rbac setup example diagrams

### CODE REVIEW

**Comment by channolanp** [comment]
Thanks for getting the graphic in & adding in icons to represent the users @tara-det-ai. Looking at the images, I just want to check if we want to change the text color on the dark theme, at least I can't see the text above the workspace.

File: docs/cluster-setup-guide/security/rbac.rst [hunk 1]
```diff
@@ -85,8 +85,20 @@ workspace.
...
+   :class: only-dark
+   :alt: Determined AI system architecture diagram dark mode
+
+.. image:: /assets/images/rbac-example-setup-light.png
+   :class: only-light
+   :alt: Determined AI system architecture diagram light mode
+
+.. note::
+
+   You can learn more about pre-canned roles such as ``Editor``, ``WorkspaceAdmin`` :ref:`in the
```
Comment (mpkouznetsov) [inline docs/cluster-setup-guide/security/rbac.rst:100]: Not to open a can of worms, and recognizing that English is not my firs language, I wonder whether "pre" in "pre-canned" is not redundant? Does adding "pre" to "canned" alter its meaning? I don't think it does.

**Review by mpkouznetsov — APPROVED** [review]
I have one question. Otherwise, looks good.

File: docs/cluster-setup-guide/security/rbac.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/cluster-setup-guide/security/rbac.rst:100]: true

**Pull request merged**